source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -216,8 +216,10 @@ - (void)addTasksToConcurrentQueue:(std::vector<std::shared_ptr<BaseCardElement>>
                     img->SetImageSize(imgSetElem->GetImageSize());
 
                     if([rendererRegistration isElementRendererOverriden:(ACRCardElementType) CardElementType::Image] == NO){
-                        /// tag a base card element with unique key
-                        [self processImageConcurrently:img];
+                    NSString *urlStr = [NSString stringWithCString:img->GetUrl().c_str()
+                                                          encoding:[NSString defaultCStringEncoding]];
+                    NSString *key = [ACRView generateKeyForElement:img];
+                    [self loadImage:urlStr key:key];
```

Comment:
Indent code.

Reply:
sorry

---
In reply to: [197509077](https://github.com/Microsoft/AdaptiveCards/pull/1641#discussion_r197509077) [](ancestors = 197509077)